docs/stdlib/string.rst

```diff
@@ -664,7 +664,7 @@ Strings
     .. code-block:: edgeql-repl
 
         db> select to_str(<datetime>'2018-05-07 15:01:22.306916-05',
-        ...               'FMDDth of FMMonth, YYYY');
+        ...               'FMDDth "of" FMMonth, YYYY');
         {'7th of May, 2018'}
         db> select to_str(<cal::local_date>'2018-05-07', 'CCth "century"');
         {'21st century'}
```

Comment:
Could we add a shorter version of the literal text note here where it's being demonstrated? Something like this maybe?
```suggestion
        {'21st century'}

    .. note::
    
        If you want to use literal text in your format string, it's best to
        enclose it in double quotes as shown above with ``of`` and
        ``century``.
```